src/styles/utils/function.scss

```diff
@@ -0,0 +1,30 @@
+@import "variables.scss";
+
+@for $index from 1 through 31 {
+  .calendar-day:nth-child(#{$index})::before {
+    content: "#{$index}";
+  }
+}
```

Comment:
into calendar block + nesting

Reply:
Why? it is a function and should be in a function block. isn't it?